images/base/files/etc/containerd/config.toml

```diff
@@ -20,3 +22,5 @@ version = 2
   tolerate_missing_hugepages_controller = true
   # explicitly use default snapshotter so we can sed it in entrypoint
   snapshotter = "overlayfs"
+  # restrict_oom_score_adj needs to be true when running inside UserNS (rootless)
+  restrict_oom_score_adj = false
```

Comment:
is there a reason to not enable it by default?

Reply:
If we enable it by default even for rootful mode, users cannot set oomScoreAdj that is stricter than containerd.